app/helpers/document_type_formatter_helper.rb

```diff
@@ -0,0 +1,22 @@
+module DocumentTypeFormatterHelper
+  CHANGES = {
+    'Aaib' => 'AAIB',
+    'Cma' => 'CMA',
+    'Dfid' => 'DFID',
+    'Esi' => 'ESI',
+    'Foi' => 'FOI',
+    'Hmrc' => 'HMRC',
+    'Html' => 'HTML',
+    'Maib' => 'MAIB',
+    'Raib' => 'RAIB',
+    'Utaac' => 'UTAAC'
+  }.freeze
+
+  def self.format(document_type)
+    document_type = document_type.try(:tr, '_', ' ').try(:capitalize)
```

Comment:
Would [`humanize`](
https://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-humanize) work here?

Reply:
and with `ActiveSupport::Inflector::Inflections#acronym` it will avoid the need for the next few lines